src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -266,7 +267,9 @@ const PIAFormPage = () => {
     const onNewPiaPage = pathname === buildDynamicPath(routes.PIA_NEW, {});
 
     // if the user is on intake or new PIA page, show the submit PIA intake button; Else submit PIA
-    if (
+    if (pia.status === PiaStatuses.FINAL_REVIEW) {
```

Comment:
Similarly, please keep anything related to status logic within the status state table. We should NOT be having this type of checks all over the place.